### ISSUE TITLE
Support updating config vars

### ISSUE THREAD

**Issue by liddellj** [geddski/grunt-release#81]
I have an existing conig var in my Gruntfile:
```
pkg: grunt.file.readJSON('package.json'),
```
I'd like to be able to have this var updated *after* the version is bumped, so that subsequent tasks which refer to pkg see the updated version. [grunt-bump](https://github.com/vojtajina/grunt-bump) supports this with a `updateConfigs` option:
```
bump: {
  options: {
    updateConfigs: ['pkg']
  }
}
```

Currently I'm using this task *and* grunt-bump to achieve what I need. If grunt-release supported an `updateConfigs` option I would not need to use grunt-bump.

**Comment by tomalec** [comment]
:+1:

**Comment by ZuBB** [comment]
also would like to see thin feature implemented (but do not have free time to do that :( )

**Comment by dorgan** [comment]
Just submitted a PR for this fix.

**Comment by drublic** [comment]
The PR just landed in master.

**Comment by dorgan** [comment]
When will this be released?

**Comment by drublic** [comment]
It's released with v0.13.0 just now.
